db/queries.py

```diff
@@ -1,46 +1,120 @@
+import init_django_orm  # noqa: F401
 from db.models import Message, User, Chat
 from django.db.models import Q, Count, F
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    """
+    Function should return list of messages that contains
```

Comment:
```suggestion
    Returns list of messages that contains
```
If you use docstring then use just present simple to describe its functionality